### ISSUE TITLE
Remote Play client performance tanks when client has Big Picture enabled

### ISSUE THREAD

**Issue by akdor1154** [ValveSoftware/steam-for-linux#7434]
#### Your system information

* Steam client version (build number or date): Oct 23 2020
* Distribution (e.g. Ubuntu): Debian 10, steam installed from debian package.
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

When client Big Picture mode is enabled on my Celeron G1610 HTPC, I get around 75% frame loss when streaming at 1080p, or around 50% frame loss at 720p.
**When client Big Picture mode is disabled, performance becomes fine**: 1-2% frameloss at 1080p.

Disabling Steam Overlay on client makes no difference, and I haven't dicked around with disabling STEAM_RUNTIME or moving `.so`s around at all (just installed steam with a fresh `.steam` dir)

Hardware acceleration via vaapi is working according to detailed stream info overlay. Graphics card is onboard Ivy Bridge, and glxinfo reports
```
Extended renderer info (GLX_MESA_query_renderer):
    Vendor: Intel Open Source Technology Center (0x8086)
    Device: Mesa DRI Intel(R) Ivybridge Desktop  (0x152)
    Version: 18.3.6
    Accelerated: yes
    Video memory: 1536MB
    Unified memory: yes
    Preferred profile: core (0x1)
    Max core profile version: 4.2
    Max compat profile version: 3.0
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 3.0
OpenGL vendor string: Intel Open Source Technology Center
OpenGL renderer string: Mesa DRI Intel(R) Ivybridge Desktop 

```

My game host/server is also Linux, same Steam version. Games tested: Dota Underlords, Artifact (maybe I should try something with a different engine just for sanity's sake...). Both client and server are on X11, not wayland.

~~Things I know I as a responsible OP need to try~~:
- [x] Steam beta on client
- [x] Steam from Valve installer, not Debian (done, with `rm ~/.steam` first)

**Comment by anclrii** [comment]
Seeing the same on a fresh install of Ubuntu 20.04 and latest steam client, also had the same issue on Debian 9 for a while.

When I start streaming the game I noticed the `UIEngineRenderT` thread starts using 50-70% cpu. Normally it's using up to 25% cpu when steam Big Picture is on the screen and none if not in BPM.

I tried to minimize steam Big Picture while streaming and that dropped `UIEngineRenderT` cpu usage and streaming was usable again. As a workaround I set a steam button chord to `minimize big picture` (power icon) for now. Every time I start streaming I press that button and streaming works great thereafter.

If I run anything locally it seems to minimize steam automatically and it would be good if steam did the same when streaming starts. I'm assuming UIEngineRenderT is rendering the BPM UI, in which case it is surprising that it's more resource intensive than streaming. Increase in cpu usage when streaming starts and UI is not on the screen seems like it may be a bug. Is this something Intel specific?

glxinfo:
```
Extended renderer info (GLX_MESA_query_renderer):
    Vendor: Intel Open Source Technology Center (0x8086)
    Device: Mesa DRI Intel(R) HD Graphics 4000 (IVB GT2) (0x166)
    Version: 20.0.8
    Accelerated: yes
    Video memory: 1536MB
    Unified memory: yes
    Preferred profile: core (0x1)
    Max core profile version: 4.2
    Max compat profile version: 3.0
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 3.0
OpenGL vendor string: Intel Open Source Technology Center
OpenGL renderer string: Mesa DRI Intel(R) HD Graphics 4000 (IVB GT2)
```
